web/ui/react-app/src/Panel.tsx

```diff
@@ -1,16 +1,6 @@
 import React, { Component } from 'react';
 
-import {
-  Alert,
-  Button,
-  Col,
-  Nav,
-  NavItem,
-  NavLink,
-  Row,
-  TabContent,
-  TabPane,
-} from 'reactstrap';
+import { Alert, Button, Col, Nav, NavItem, NavLink, Row, TabContent, TabPane } from 'reactstrap';
```

Comment:
Not sure if we need this multi- to single-line reformatting of import statements. Maybe better to leave this decision to the developer depending on what they find more reasonable on a case-by-case basis? If it's hard to turn this off individually though, it's ok too.

Reply:
Agreed I'd rather leave it up to the developer.

Reply:
After looking into it seems that you can't tell it not too, without putting a `//prettier-ignore` comment in front of it which is a bit nasty.

Reply:
@CSTDev It's probably due to their stance on configurability (https://prettier.io/docs/en/option-philosophy.html). I guess it's fine then, we can live with it.